OpenSearch: add dedicated user for Filebeat (#3079) (#3221)
epiphany-platform/epiphany: docs/home/howto/MONITORING.md

By default OpenSearch Dashoboards adjusts the UTC time in `@timestamp` to the browser's local timezone. This can be changed in `Stack Management` > `Advanced Settings` > `Timezone for date formatting`.

## How to configure default passwords for service users in OpenSearch Dashboards, OpenSearch and Filebeat

> Epiphany provides two roles that inclue OpenSearch installation: `logging` (includes OpenSearch-Dashboards) and `opensearch`.

Reviewer: ```suggestion
Epiphany provides two components that include OpenSearch installation: `logging` (by default includes OpenSearch-Dashboards as well) and `opensearch`.
```

> In order to learn more about both roles, please look through documentation:

Reviewer: ```suggestion
In order to learn more about both components, please look through documentation:
```

- [logging](./LOGGING.md#centralized-logging-setup)
- [opensearch](./DATABASES.md#how-to-start-working-with-opensearch)

To configure admin password for OpenSearch Dashoboards ( previously Kibana ) and OpenSearch you need to follow the procedure below.

### Logging component

#### Logging role

Default users configured by Epiphany for `logging` role are:

> - `kibanaserver`<sup>[1]</sup> user (needed by default Epiphany installation of Dashboards )

Reviewer: ```suggestion
- `kibanaserver`<sup>[1]</sup> - needed by default Epiphany installation of Dashboards
```

> - `filebeatservice` user (needed by default Epiphany installation of Filebeat)

Reviewer: ```suggestion
- `filebeatservice` - needed by default Epiphany installation of Filebeat
```

  [...]
  admin_password: YOUR_PASSWORD
  kibanaserver_password: YOUR_PASSWORD
  filebeatservice_password: PASSWORD_TO_CHANGE
  demo_users_to_remove:
  - kibanaro
  - readall
  - logstash
  - snapshotrestore
```

### OpenSearch component

Default user provided by Epiphany for OpenSearch role is `admin`. Additionally, Epiphany removes all demo users except `admin` user.
Those users are listed in `demo_users_to_remove` section of `configuration/opensearch` manifest doc ( see example below ).
To change `admin` user's password, change value for the `admin_password` key.

**We strongly advice to set different password for admin user.**

> Note that adding `opensearch-dashboards` mapping in `configuration/feature-mappings` under `opensearch` component requires commenting out `kibanaserver` user form `demo_users_to_remove` section (as presented in configuration below). This step should be followed by changing default password for `kibanaserver` user by modifying value for `kibanaserver_password` key.

Reviewer: ```suggestion
Note that adding `opensearch-dashboards` mapping in `configuration/feature-mappings` under `opensearch` component requires commenting out `kibanaserver` user in `demo_users_to_remove` section (as presented in configuration below). This step should be followed by changing default password for `kibanaserver` user by modifying value for `kibanaserver_password` key.
```
